iron-fit-behavior.html

```diff
@@ -111,7 +179,40 @@
       return fitTop;
     },
 
+    /**
+     * The element that should be used to position the element when
+     * it opens, if no position target is configured.
+     */
+    get _defaultPositionTarget() {
+      var parent = Polymer.dom(this).parentNode;
+
+      if (parent.nodeType === Node.DOCUMENT_FRAGMENT_NODE) {
+        parent = parent.host;
+      }
+
+      return parent;
+    },
+
+    /**
+     * The horizontal align value, accounting for the RTL/LTR text direction.
+     */
+    get _localeHorizontalAlign() {
+      if (this._isRTL) {
+        // In RTL, "left" becomes "right".
```

Comment:
FWIW this code (and alignment feature as a whole) was ported from `iron-dropdown` so that all overlays can benefit from it.
`iron-dropdown` was already doing this auto-flipping before the port (introduced in https://github.com/PolymerElements/iron-dropdown/pull/42 after the request you made in https://github.com/PolymerElements/iron-dropdown/issues/29)